catalyst/dl/callbacks/logging.py

```diff
@@ -183,6 +186,92 @@ def on_stage_end(self, state: RunnerState):
             logger.close()
 
 
+class TelegramLogger(Callback):
+    """
+    Logger callback, translates state.metrics to telegram channel
+    """
+
+    def __init__(
+        self,
+        token: str,
```

Comment:
It is very useful to add all your experiment files into git (including configs), and at the same time, you don't really want to add any secrets into git. So, in my opinion, it is better to read `chat_id` and `token` from env variables or something like `telegram.cfg`.

Reply:
I'm sure that this feature (configuration parsing) must be not it telegram logger itself. Configuration manager (like in various .net or java frameworks) or global configuration (like in Flask), or catalyst.global.config['telegram_chat_id']  - something this.